packages/components/src/composite/current/stories/index.story.tsx

```diff
@@ -11,35 +11,31 @@ import { isRTL } from '@wordpress/i18n';
 /**
  * Internal dependencies
  */
-import {
-	Composite,
-	CompositeGroup,
-	CompositeRow,
-	CompositeItem,
-	useCompositeStore,
-} from '..';
+import * as Composite from '..';
 import { UseCompositeStorePlaceholder, transform } from './utils';
 
 const meta: Meta< typeof UseCompositeStorePlaceholder > = {
-	title: 'Components/Composite (V2)',
+	title: 'Components/Composite',
 	component: UseCompositeStorePlaceholder,
 	subcomponents: {
 		// @ts-expect-error - See https://github.com/storybookjs/storybook/issues/23170
-		Composite,
+		Root: Composite.Root,
 		// @ts-expect-error - See https://github.com/storybookjs/storybook/issues/23170
-		CompositeGroup,
+		Group: Composite.Group,
 		// @ts-expect-error - See https://github.com/storybookjs/storybook/issues/23170
-		CompositeRow,
+		Row: Composite.Row,
 		// @ts-expect-error - See https://github.com/storybookjs/storybook/issues/23170
-		CompositeItem,
+		Item: Composite.Item,
 	},
-	tags: [ 'status-private' ],
 	parameters: {
 		docs: {
 			canvas: { sourceState: 'shown' },
 			source: { transform },
 			extractArgTypes: ( component: React.FunctionComponent ) => {
-				const name = component.displayName;
+				const name =
+					component.displayName === 'useStore'
+						? 'useCompositeStore'
+						: component.displayName;
```

Comment:
Necessary to generale the correct documentation for the `Composite.useStore` hook.